src/models.rs

```diff
@@ -205,6 +226,28 @@ impl TimeEntry {
             format!("[{}]", self.tags.join(", "))
         }
     }
+
+    pub fn as_network_time_entry(&self) -> NetworkTimeEntry {
+        NetworkTimeEntry {
+            id: self.id,
+            description: self.description.to_string(),
+            start: self.start,
+            stop: self.stop,
+            duration: self.duration,
+            billable: self.billable,
+            workspace_id: self.workspace_id,
+            tags: self.tags.clone(),
+            project_id: match self.project.clone() {
```

Comment:
```suggestion
            project_id: match self.project.clone() {
```
Maybe just map, instead of `match`